cdap-common/src/main/java/co/cask/cdap/common/kerberos/SecurityUtil.java

```diff
@@ -167,4 +175,38 @@ public void run() {
         }, delaySec, delaySec, TimeUnit.SECONDS);
     }
   }
+
+  /**
+   * Helper to create a {@link KerberosPrincipal} from given principal string.
+   * <p>
+   * Supports two Kerberos name types:
+   * <ul>
+   * <li>KRB_NT_PRINCIPAL:  Just the name of the principal as in DCE, or for users. For example: alice@REALM</li>
+   * <li>KRB_NT_SRV_HST:  Service with host name as instance(telnet, rcommands).
```

Comment:
its from the description of the name type whose link is below in the documentation.